### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by pancakereport** [berkeley-scf/stat999-myst#13]
Hey @ryanlovett I added the axe-core a11y checking action to this repo and confirmed that everything works as expected. There's a few failures:
* color contrast (home page)
* nested interactive (home page)
* iframe associated issues (calendar)

Color contrast and nested interactive should be easy to take care of. The iframe issues we addressed in https://github.com/berkeley-cdss/berkeley-class-site with a full-calendar integration (no more iframe). It looks like this template is using the iframe embed provided with MyST. Maybe we could make some contributions upstream to improve accessibility here, but I'm not sure that will work because it's embedding a problematic thing to begin with. (I've had problems with calendar and YouTube video iframes and am just starting to think about what to do). We could also stop axe from flagging the specific issues with iframes (so the tests pass, but it's won't address the issue). 

I would like to hear from you about whether you think we should make the template a solid boiler plate (by, for example, making sure all color defaults meet color contrast standards) or just address the issues flagged by the [deployed template](https://berkeley-cdss.github.io/course-site-myst/). And I'm just filing the issue for tracking, I might address some of this at some point.
